config/detekt.yml

```diff
@@ -126,7 +126,7 @@ complexity:
   LongParameterList:
     active: true
     functionThreshold: 6
-    constructorThreshold: 7
+    constructorThreshold: 10
```

Comment:
Might we create code style rules to prevent inconsistent code to be applied, instead of adopting the rules when inconsistent code written?

Reply:
changed to 8, but i if i need more values in constructor, how can i improve it to make less args in this case?